### PR TITLE
Repository Migration

### PR DESCRIPTION
**Pull request by lukesneeringer**
I think all of us know _exactly_ what this does by now. ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,151 @@
-# Node.js Client for Google Cloud Speech API ([Beta](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-[Google Cloud Speech API][Product Documentation]: Google Cloud Speech API.
-- [Client Library Documentation][]
-- [Product Documentation][]
+# [Google Cloud Speech API: Node.js Client](https://github.com/GoogleCloudPlatform/google-cloud-node)
```
Comment (stephenplusplus) [inline README.md]: Old link to the GCN repo.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,151 @@
...
 
-## Quick Start
-In order to use this library, you first need to go through the following steps:
+[![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![CircleCI](https://img.shields.io/circleci/project/github/GoogleCloudPlatform/google-cloud-node.svg?style=flat)](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node)
```
Comment (stephenplusplus) [inline README.md]: These shields all reference GCN.

File: README.md [hunk 3]
```diff
@@ -1,59 +1,151 @@
...
 
-### Installation
-```
-$ npm install --save @google-cloud/speech
-```
+The [Cloud Speech API](https://cloud.google.com/speech/docs) enables easy integration of Google speech recognition technologies into developer applications. Send audio and receive a text transcription from the Cloud Speech API service.
+
+
+* [Speech API Node.js Client API Reference][client-docs]
+* [github.com/GoogleCloudPlatform/google-cloud-node](https://github.com/GoogleCloudPlatform/google-cloud-node)
```
Comment (stephenplusplus) [inline README.md]: Probably meant to be a link to this repo.

File: README.md (hunk 1 above)
Comment (jmdobry) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (jmdobry) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (jmdobry) [reply README.md]: Fixed

**Pull request merged**